src/elevator/fsm.rs

```diff
@@ -213,6 +249,16 @@ impl ElevatorFSM {
     fn handle_event(&mut self, event: Event) {
         match event {
             Event::FloorReached(floor) => {
+                //Resting timer for drive time between floors
+                if self.started_moving && !self.peer_enable{
```

Comment:
This is behind main, handle_event isn't a function anymore. Must be moved into handle_floor_hit function.